python/nav/bitvector.py

```diff
@@ -31,7 +33,7 @@ class BitVector(object):
     def __init__(self, octetstring):
         self.vector = array.array("B", octetstring)
 
-    def __str__(self):
+    def to_bytes(self):
```

Comment:
Actually, upon testing, this breaks BitVector. Just a few lines down, `BitVector.__repr__()` uses `str()` on itself as part of the output, which appears to turn into an infinite recursion (under Python 2)..